### PR TITLE
remove java dependency in favor of Rcpp-made RPostgres

### PR DESCRIPTION
**Pull request by kirillseva**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: berdie
 Type: Package
 Title: Berdie
-Version: 0.0.2
+Version: 0.0.3
```
Comment (peterhurford) [inline DESCRIPTION]: Seems like a major version up to me.

**Comment by bruno615** [comment]
Nice!

File: DESCRIPTION (hunk 1 above)
Comment (kirillseva) [inline DESCRIPTION]: i feel guilty. hadley has done all the work, so I felt like I should just make a tiny bump

Comment (peterhurford) [inline DESCRIPTION]: It's a pretty large change to the package.  Also, you should feel good; I've found it difficult to integrate Postgres libraries in the past.

**Comment by robertzk** [comment]
Make sure to remove the JAR in `vendor/jars` as well.

**Comment by robertzk** [comment]
:+1:

**Comment by kirillseva** [comment]
Merge?

**Pull request merged**